longevity_test.py

```diff
@@ -462,13 +462,9 @@ def _flush_all_nodes(self):
 
     def get_email_data(self):
         self.log.info("Prepare data for email")
-        email_data = {}
         grafana_dataset = {}
 
-        try:
-            email_data = self._get_common_email_data()
-        except Exception as error:  # pylint: disable=broad-except
-            self.log.exception("Error in gathering common email data: Error:\n%s", error, exc_info=error)
+        email_data = self._get_common_email_data()
```

Comment:
This change is about changing logic , not just lint.
Is it intentional?

Reply:
yes, it's the same across all tests, and in majority of them it's not under try/except, the all to this function `get_email_data()` is also wrapper with try/except clause.
anyhow there no point moving one, if we didn't had this minimal set of information, we won't be able to format the email report out of it at this point.

we should be raising events on those failures, not printing it to the log only.